### DISCUSSION TITLE
archery是开源的吗？

### DISCUSSION THREAD

**Discussion by yikebantang** [hhyo/Archery#2453] [Q&A]
下载Archery有一周多的时间了，部署好之后一直没用，等到用的时候发现web报“服务异常，请联系管理员”，查看日志发现是数据库连接不上，我以为是我部署的问题，一直在排查错误，然后等我登录数据库时发现库里只剩一个表，查看这个表发现以下内容：
| I have backed up all your databases. To recover them you must pay 0.0117 BTC (Bitcoin) to this address: 1L6N9twGfwvXqvmWUf4GyiGsn9fw6LKz6f . Backup List: archery, recover_your_data. After your payment email me at sqlrecover471@onionmail.org with your server IP (*) and transaction ID and you will get a download link to your backup. Emails without transaction ID and server IP will be ignored.  | 1L6N9twGfwvXqvmWUf4GyiGsn9fw6LKz6f |

我想问是我包下载的有问题还是服务本身就是这样的

**Comment by LeoQuote** [comment]
哥，你被黑客入侵了，记得从GitHub官网下安装包，做好服务器的安全防护

**Reply by yikebantang** [reply]
我是在https://github.com/hhyo/Archery/releases/tag/v1.10.0这个连接下的
